### ISSUE TITLE
Open source alternative to VB-Cable

### ISSUE THREAD

**Issue by antiboredom** [antiboredom/zoom-escaper#2]
would love to find an open source alternative to VB Cable...

**Comment by q2dg** [comment]
Maybe Pipewire could do the job... Meanwhile, there is PulseAudio (see https://askubuntu.com/questions/257992/how-can-i-use-pulseaudio-virtual-audio-streams-to-play-music-over-skype#258369 or https://endless.ersoft.org/pulseaudio-loopback)

**Comment by antiboredom** [comment]
@q2dg are these cross platform? Ideally I'd replace VB-Cable with something something that works on all platforms...

**Comment by q2dg** [comment]
Unfortunately, they aren't. Both PulseAudio (current technology) and Pipewire (future technology) are tied to Linux systems, sorry

**Comment by mhalano** [comment]
Yeah, that's a bummer. In my company (best place to use it) we use Linux a lot.
